scripts/jest.setup.ts

```diff
@@ -1,4 +1,4 @@
 import { configure } from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
+import ReactSixteenAdapter from 'enzyme-adapter-react-16'
```

Comment:
I would name it EnzymeAdapter

Reply:
Thanks. ReactSixteenAdapter is prefer (if esModuleInterop: false in tsconfig).